src/bindgen/utilities.rs

```diff
@@ -146,10 +146,9 @@ impl_syn_item_helper!(syn::ItemTrait);
 impl_syn_item_helper!(syn::ItemImpl);
 impl_syn_item_helper!(syn::ItemMacro);
 impl_syn_item_helper!(syn::ItemMacro2);
-impl_syn_item_helper!(syn::ItemExistential);
 impl_syn_item_helper!(syn::ItemTraitAlias);
 
-impl SynItemHelpers for syn::ItemVerbatim {
+/*impl SynItemHelpers for syn::ItemVerbatim {
```

Comment:
If this no longer exists the code should be removed.

Reply:
Woops... Forgot to remove that